class/deeplink-schema-match.ts

```
@@ -450,8 +462,8 @@ class DeeplinkSchemaMatch {
     return undefined;
   }
 
-  static bip21decode(uri) {
-    if (!uri) return {};
+  static bip21decode(uri: string) {
+    // if (!uri) return {};
```

hmm, this guard clause was here for a reason, probably

---

was  added here:  https://github.com/BlueWallet/BlueWallet/commit/6565fd531dab5b8757061151e615ed0fcb4bd709

---

I'm not a fan of "return something simular" in case of error. I would rather throw an error in this case